src/UglyToad.PdfPig.Tokens/ArrayToken.cs

```diff
@@ -88,5 +88,26 @@ public override string ToString()
 
             return builder.ToString();
         }
+        
+        /// <inheritdoc />
+        public bool Equals(IToken obj)
+        {
+            if (this == obj)
+                return true;
```

Comment:
Sorry I realise I never got round to sharing the dotsettings I'm using. In general I'd like to keep to having the curly braces even for single line statements. It's more verbose but also more explicit about what's in scope and stops problems in future maintenance where things might get inadvertently added/removed from a scope. This is a comment across the modified files but I'll just highlight it here.